internal/image/image.go

```diff
@@ -73,6 +69,172 @@ func ScanImage(r reporter.Reporter, imagePath string) (ScanResults, error) {
 	return scannedLockfiles, err
 }
 
+type Image struct {
+	flattenedFileMaps []FileMap
```

Comment:
nit: how about `flattenedLayers`? It's not immediately obvious this is referring to layers from the name otherwise.

Reply:
And also per https://google.github.io/styleguide/go/decisions#repetitive-with-type